### PR TITLE
Indicate that state is optional for OIDC logout

### PR DESCRIPTION
**Pull request by orenyk**
Resolves LG-7684

Do not merge before https://github.com/18F/identity-idp/pull/7045 is deployed (likely Monday)

### CODE REVIEW

File: _pages/oidc.md [hunk 1]
```diff
@@ -436,7 +436,7 @@ To log out a user, send them to the `/openid_connect/logout` endpoint with the f
 - **post_logout_redirect_uri**
   The URI Login.gov will redirect to after logout. **This must also be registered with the Login.gov IdP in advance.**
 
-- **state**
+- **state** (optional)
   A unique value at least 22 characters in length used for maintaining state between the request and the callback. This value will be returned to the client on a successful logout.
```
Comment (pauldoomgov) [inline _pages/oidc.md:440]: Is it worth spelling out how this is used a BIT while we are here?   Maybe "This value will be returned to the client on a successful logout as a parameter of `state` added to the `post_logout_redirect_uri` making it visible to the SP."

**Review by pauldoomgov — APPROVED** [review]
LGTM!  One minor thought you can take or leave.

File: _pages/oidc.md (hunk 1 above)
Comment (orenyk) [reply _pages/oidc.md:440]: sure! more clarity is always better 😄

**Comment by orenyk** [comment]
We'll merge this on Monday after the deployment

**Pull request merged**